examples/agent.py

```diff
@@ -155,6 +157,12 @@ async def run(self, task: str, url: str | None = None) -> AgentOutput:
         3. When and how to determine task completion
         4. Error handling and recovery strategies
         """
+        credentials, password = get_credentials_for_domain(clean_url(url) if url else None)
+        if credentials and password:
+            logger.info(f"🔑 Using credentials for {url}: {credentials} / {password}")
```

Comment:
I don't we don't want the credentials to be logged. You can just say `Found credentials for URL` or something like that

Reply:
Actually I think the logging should only be `Using credential vault` and only when a `login action` is indeed needed then you can have a logging message in the env